test_all_examples.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+import test_example_repo as run
+
+def test_all_examples():
+	rootPath = os.getcwd()
+	run.test_example_repo("typescript", "crud-app", rootPath)
+	run.test_example_repo("typescript", "getting-started", rootPath)
+	run.test_example_repo("typescript", "hello-world", rootPath)
+	run.test_example_repo("typescript", "multiversx", rootPath)
+	run.test_example_repo("typescript", "quiz-app-prisma", rootPath)
+	run.test_example_repo("typescript", "shopping-cart", rootPath)
+	run.test_example_repo("typescript", "todo-list-angular", rootPath)
+	run.test_example_repo("typescript", "todo-list-flutter", rootPath)
+	run.test_example_repo("typescript", "todo-list", rootPath)
+	run.test_example_repo("typescript", "webhook", rootPath)
```

Comment:
1. variables in python are `snake_case`
2. it's better to define an array of examples and then loop through them, than copy paste the function multiple times.

```suggestion
    root_path = os.getcwd()
    examples = [
        { language: "typescript", app: "crud-app" },
        { language: "typescript", app: "getting-started" },
        { language: "typescript", app: "hello-world" },
        { language: "typescript", app: "multiversx" },
        { language: "typescript", app: "quiz-app-prisma" },
        { language: "typescript", app: "shopping-cart" },
        { language: "typescript", app: "todo-list-angular" },
        { language: "typescript", app: "todo-list-flutter" },
        { language: "typescript", app: "todo-list" },
        { language: "typescript", app: "webhook" },
    ]

    for example in examples:
        run.test_example_repo(example.language, example.app, root_path)
```